.github/workflows/terraform-plan-datadog-v1.yml

```diff
@@ -0,0 +1,107 @@
+name: Terraform plan for Datadog resources
```

Comment:
What's the use-case for this workflow 🤔

Reply:
ah sorry it was test I didn't mean to commit it (yet)